api/swagger.yml

```diff
@@ -2734,6 +2746,47 @@ paths:
         default:
           $ref: "#/components/responses/ServerError"
 
+  /repositories/{repository}/branches/{branch}/cherry-pick:
+    parameters:
+      - in: path
+        name: repository
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: branch
+        required: true
+        schema:
+          type: string
+    post:
+      tags:
+        - branches
+      operationId: cherryPick
+      summary: Cherry-Pick the given reference commit into the given branch
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CherryPickCreation"
+      responses:
+        204:
```

Comment:
201 Created? (Whatever we do for merge / commit should work here, but I'd guess 201 because a POST was successful.)